### PR TITLE
Add Status route

### LINKED ISSUE #106: Add status get and post method
Good to have a status get method. 

This will be handy for displaying to the user at some point

We could have:
- {'status':'ok','message=''}
- {'status'error': 'message', 'Satellite data is down'}
- {'status':'warning', 'Satellite data is down, so using model v1.01 instead'}

This would involve making a 
## Model 
1. Pydantic model / sqlmodel in `nowcatsing_datamodel'
```

class StatusSQL(CreatedMixin):
    """ffffffff"""

    __tablename__ = "status"

    id = Column(Integer, primary_key=True)
    status = Column(String)
    message = Column(String)

class Status(EnhancedBaseModel):
    """"""

    status: string = Field(
        ...,
        description="ggggg",
    )
    message: str = Field(
        ..., description="The forecasted value in MW"
    )
 
+ validate on status in 'ok', 'warning', 'error'

```

**tests**
- check status object can be made, and move to pydantic object
- check validation makes error when status is 'laugh'

## Read function
2. create a read last function in `nowcatsing_datamodel' - see other read [examples](https://github.com/openclimatefix/nowcasting_datamodel/blob/main/nowcasting_datamodel/read/read.py#L28)

**tests**
- make 2 lots dummy data and check last one is read

## API
3. create api end point to expose this in `nowcatsing_api'

**tests**
- add dummy data, and check end points works

### PR DESCRIPTION
**Pull request by jacobbieker**
# Pull Request

## Description

Adds the status route to the webapp

Depends on https://github.com/openclimatefix/nowcasting_datamodel/pull/55 being merged for the datamodel

Fixes #106 

## How Has This Been Tested?

Unit tests

- [x] Yes

_If your changes affect data processing, have you plotted any changes? i.e. have you done a quick sanity check?_

- [ ] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/tests/test_status.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    """Check main GB/pv/status route works"""
+    status = Status(message="Good", status="ok").to_orm()
+    db_session.add(status)
+
+    app.dependency_overrides[get_session] = lambda: db_session
+
+    response = client.get("/v0/GB/solar/status")
+    assert response.status_code == 200
+
+    _ = Status(**response.json())
```
Comment (peterdudfield) [inline src/tests/test_status.py:21]: perhaps just check, that the return object, has the same message

**Review by peterdudfield — APPROVED** [review]
Looks great, one small comment, and we cant deploy until ive done
https://github.com/openclimatefix/nowcasting_datamodel/issues/57

**Comment by jacobbieker** [comment]
> > Looks great, one small comment, and we cant deploy until ive done [openclimatefix/nowcasting_datamodel#57](https://github.com/openclimatefix/nowcasting_datamodel/issues/57)
> 
> this is now done

Awesome! Great, I'll merge shortly then

**Comment by peterdudfield** [comment]
Ive added one value to the new table
Screenshot 2022-05-25 at 15 22 49

**Comment by peterdudfield** [comment]
> > > Looks great, one small comment, and we cant deploy until ive done [openclimatefix/nowcasting_datamodel#57](https://github.com/openclimatefix/nowcasting_datamodel/issues/57)
> > 
> > 
> > this is now done
> 
> Awesome! Great, I'll merge shortly then

feel free to deploy, and then try it out. If it doesnt work, then you can role back

**Pull request merged**